src/typed-error.ts

```diff
@@ -0,0 +1,29 @@
+class BaseError {
+    constructor () {
```

Comment:
NIT: I think there's no need for space before the parens, essentially it's a special named function